internal/deepdoc/parser/pdf/inference/client_dla_garbage_test.go

```
@@ -0,0 +1,61 @@
+// implement the gate; it becomes the regression test once the gate is added
+// (mirror layout_recognizer.__call__:97 inside Client.DLA, or inside the
+// backend's forward).
+func TestDeepDocHTTP_DLA_DropsLowConfidenceGarbageLayout(t *testing.T) {
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.Path != "/predict/dla" {
+			t.Errorf("path = %q, want /predict/dla", r.URL.Path)
+		}
+		// bboxes = [[x0, y0, x1, y1, confidence, classId], ...]
+		// classId 2 = "reference" (a garbage layout), conf 0.30 < 0.4 -> dropped
+		// classId 1 = "text" (not a garbage layout), conf 0.90 -> kept
+		json.NewEncoder(w).Encode(map[string]any{
```

golangci-lint (2.12.2)

[error] Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)